app/templates/components/addon-tabs.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="addon-tabs">
+  <div class="addon-tabs--header">
+    <button
+      class="addon-tabs--tab {{if (eq this.currentTab 0) "active-tab"}}"
+      onclick={{action (mut this.currentTab) 0}}
+    >
+      ember-cli-sass
+    </button>
+    <button
+      class="addon-tabs--tab {{if (eq this.currentTab 1) "active-tab"}}"
+      onclick={{action (mut this.currentTab) 1}}
+    >
+      SVG jar
+    </button>
+    <button
+      class="addon-tabs--tab {{if (eq this.currentTab 2) "active-tab"}}"
+      onclick={{action (mut this.currentTab) 2}}
+    >
+      Percy
```

Comment:
We should not include percy because it requires payment. There is another open source addon that does visual regression testing if you want to find and use that.

Reply:
They do have a quite generous free plan, though: https://percy.io/pricing . Up to 10 team members and 5000 snapshots/month.

Reply:
I avoided making any decision on this point because actually it's covered by https://github.com/ember-learn/ember-website/issues/521 

I'm splitting this into it's own PR so we can decide if that PR will be blocked until #521 is finished and the content included in the same PR, or if it can be a follow-on PR 👍